envs/example/ci/group_vars/all.yml

```diff
@@ -13,3 +13,11 @@ ceilometer:
   logging:
     debug: True
     verbose: True
+
+keystone:
+    multi_domain:
```

Comment:
Given that the only configuration we're supporting for this is ldap, maybe we should just change the flag name to `ldap_domain`, or `ldap_user_domain`?

Dunno, just seems too generic, but, bike shed.